### PR TITLE
Remove unused Javascript/Node dependencies

### PR DESCRIPTION
**Pull request by danlipert**
##### Description

This PR removes unused Javascript/node dependencies.

##### Checklist

- [x] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [x] Linter status: 100% pass
- [x] Added relevant tests
- [x] Tests pass and code coverage has not decreased
- [x] Changes don't break existing behavior
- [x] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [x] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

##### Affected core subsystem(s)

Just dependencies

##### Refers/Fixes

Refs: https://github.com/gitcoinco/web/issues/3668

###### Contributor

- [x] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Tested all changes **locally**
- [x] Verified existing functionality
- [x] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

###### Funder

- [ ] Validated requested changes were made to specification
- [ ] Bounty payout released to the contributor

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,25 +21,9 @@
   },
   "homepage": "https://github.com/gitcoinco/web#readme",
   "devDependencies": {
-    "babel": "^6.23.0",
-    "babel-core": "^6.26.3",
     "babel-eslint": "^8.2.6",
```
Comment (octavioamu) [inline package.json]: I think this one is not used

File: package.json [hunk 2]
```diff
@@ -21,25 +21,9 @@
...
-    "eslint-plugin-node": "^6.0.1",
-    "eslint-plugin-promise": "^3.8.0",
-    "eslint-plugin-standard": "^3.1.0",
     "stylelint": "^8.4.0",
-    "stylelint-config-recommended": "^2.1.0",
-    "stylelint-webpack-plugin": "^0.9.0",
-    "webpack": "^3.12.0",
-    "webpack-bundle-tracker": "^0.3.0",
-    "webpack-dev-server": "^2.11.3"
+    "stylelint-config-recommended": "^2.1.0"
```
Comment (octavioamu) [inline package.json]: Also this one I think is not used

**Comment by thelostone-mc** [comment]
Apart from @octavioamu comment -> i think we are good ^_^

**Comment by thelostone-mc** [comment]
Apart from @octavioamu comment on `babel-eslint` -> i think we are good ^_^

**Comment by octavioamu** [comment]
@thelostone-mc My comment was wrong, in the config `.eslintrc ` is requiring that packages. So ready to Merge!

**Comment by danlipert** [comment]
@octavioamu @thelostone-mc I've got one more change incoming - was able to remove the babel-eslint library by bumping the eslint ecmascript version 8/2017 :+1: will push commit shortly

File: .eslintrc [hunk 3]
```diff
@@ -13,10 +13,9 @@
     "$": true
   },
   "plugins": [],
-  "parser": "babel-eslint",
   "parserOptions": {
     "sourceType": "module",
-    "ecmaVersion": 6,
+    "ecmaVersion": 8,
```
Comment (thelostone-mc) [inline .eslintrc]: ❤️

**Comment by danlipert** [comment]
@thelostone-mc Nope! I think thats it 👍

**Pull request merged**